From 2250f74b8214e848986d44cfba2f2b551cdee1a5 Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Wed, 4 Oct 2023 17:45:36 +0200
Subject: [PATCH 043/184] 
 linux-mainline-and-mali-allwinner-h6-kernel/misc.616/patches/fix-h618-eth-hack.patch

---
 drivers/clk/sunxi-ng/ccu-sun50i-h616.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/clk/sunxi-ng/ccu-sun50i-h616.c b/drivers/clk/sunxi-ng/ccu-sun50i-h616.c
index ad4ad6be0..9dfb787ee 100644
--- a/drivers/clk/sunxi-ng/ccu-sun50i-h616.c
+++ b/drivers/clk/sunxi-ng/ccu-sun50i-h616.c
@@ -363,7 +363,7 @@ static SUNXI_CCU_GATE(bus_dbg_clk, "bus-dbg", "psi-ahb1-ahb2",
 static SUNXI_CCU_GATE(bus_psi_clk, "bus-psi", "psi-ahb1-ahb2",
 		      0x79c, BIT(0), 0);
 
-static SUNXI_CCU_GATE(bus_pwm_clk, "bus-pwm", "apb1", 0x7ac, BIT(0), 0);
+// static SUNXI_CCU_GATE(bus_pwm_clk, "bus-pwm", "apb1", 0x7ac, BIT(0), 0);
 
 static SUNXI_CCU_GATE(bus_iommu_clk, "bus-iommu", "apb1", 0x7bc, BIT(0), 0);
 
@@ -770,7 +770,7 @@ static struct ccu_common *sun50i_h616_ccu_clks[] = {
 	&avs_clk.common,
 	&bus_dbg_clk.common,
 	&bus_psi_clk.common,
-	&bus_pwm_clk.common,
+// 	&bus_pwm_clk.common,
 	&bus_iommu_clk.common,
 	&dram_clk.common,
 	&mbus_dma_clk.common,
@@ -903,7 +903,7 @@ static struct clk_hw_onecell_data sun50i_h616_hw_clks = {
 		[CLK_AVS]		= &avs_clk.common.hw,
 		[CLK_BUS_DBG]		= &bus_dbg_clk.common.hw,
 		[CLK_BUS_PSI]		= &bus_psi_clk.common.hw,
-		[CLK_BUS_PWM]		= &bus_pwm_clk.common.hw,
+// 		[CLK_BUS_PWM]		= &bus_pwm_clk.common.hw,
 		[CLK_BUS_IOMMU]		= &bus_iommu_clk.common.hw,
 		[CLK_DRAM]		= &dram_clk.common.hw,
 		[CLK_MBUS_DMA]		= &mbus_dma_clk.common.hw,
-- 
2.39.2

